queries/lics_with_primary_users.md

```diff
@@ -0,0 +1,36 @@
+# Licences with a primary user
+
+- **Business**
+- **2023-06-27**
+- [WATER-4047](https://eaflood.atlassian.net/browse/WATER-4047)
+
+> [We need to] monitor the impact the new roles are having on Registration etc in each area[. N]ow we have no/limited location data but [we believe we] can use the licence numbers to merge with NALD data in Power BI and pull together the benchmarks.
```

Comment:
There seem to be some odd bits of text wrapped in []. Guess you didn't mean to do that?

Reply:
> When writers insert or alter words in a direct quotation, square brackets—[ ]—are placed around the change. The brackets, always used in pairs, enclose words intended to clarify meaning, provide a brief explanation, or to help integrate the quote into the writer’s sentence. 

https://writingcommons.org/article/inserting-or-altering-words-in-a-direct-quotation/

So yep, I meant to do it! 😁

Reply:
I never knew that, thanks. Guess I better approve it then! :grin: